shell/imports/client/apps/applist-client.js

```diff
@@ -226,6 +213,14 @@ Template.sandstormAppListPage.events({
     });
   },
 
+  "click .install-button": function(event) {
+    event.preventDefault();
+    event.stopPropagation();
+    const url = event.currentTarget.getAttribute("href");
+    window.open(url +
```

Comment:
Don't you need to add `?host=` in here?

Reply:
I don't think so, I think he is grabbing the URL wholesale from the HTML side of this.

Reply:
OK, then isn't this adding a second copy of the Sandstorm server URL, since the `?host=https://example.com` is already in there? You'd end up with `?host=https://example.comhttps://example.com`...

Reply:
Ahhhh, I see what you mean now. Yeah, this would probably cause that bug here.